.changes/v3.6.0/795-improvements.md

```diff
@@ -0,0 +1 @@
+* Update codebase to be compatible with VCD v35.0 [GH-795]
```

Comment:
Let's provide a bit more context:

```suggestion
* Update codebase to be compatible with changes in go-vcloud-director due to bump to VCD API V35.0 [GH-795]
```

Please also add notes to PR description/merge commit message about changes in schema.Schema and diag usage.

Reply:
Changes done!
Also, I'll write in the merge commit the changes you suggest. Thanks!